znoyder/config.yml

```diff
@@ -7,36 +7,39 @@ collect:
   'openstack-tox-py38': 'osp-tox-py38'
   'openstack-tox-py39': 'osp-tox-py39'
 
-excluded_jobs: {}
-
-excluded_jobs_by_tag:
-  'osp-17.0':
-    'openstack-tox-py36': 'OSP 17 release is not using py38 and lower'
-    'openstack-tox-py37': 'OSP 17 release is not using py38 and lower'
-    'openstack-tox-py38': 'OSP 17 release is not using py38 and lower'
-
-excluded_jobs_by_project_and_tag:
+exclude:
+  '/.*/':
+    'osp-17.0':
+      'openstack-tox-py36': 'OSP 17 release is not using py38 and lower'
+      'openstack-tox-py37': 'OSP 17 release is not using py38 and lower'
+      'openstack-tox-py38': 'OSP 17 release is not using py38 and lower'
   'nova':
     'osp-17.0':
       'openstack-tox-py36': 'we do not support this Python version'
       'openstack-tox-py38': 'we do not support this Python version'
 
-additional_jobs: {}
-
-additional_jobs_by_tag:
-  'osp-17.0':
-    'openstack-tox-py39':
-      'voting': false
-      'type': 'gate'
-
-additional_jobs_by_project_and_tag:
+include:
+  '/.*/':
+    'osp-17.0':
+      'openstack-tox-py39':
+        'voting': false
+        'type':
```

Comment:
I know it wasn't introduced by this PR, but I would change 'type' to 'pipeline'. Better stick to the same terminology, rather than creating a separate one in Znoyder

Reply:
Hmm, I agree with you, this comes from the our early days. The thing is that it is related to ZuulJob implementation in znoyder/lib/zuul.py, so I propose to keep it as-is right now here and introduce further change (with dedicated Jira task) to redesign this starting from our Zuul lib.

Reply:
Task created: https://issues.redhat.com/browse/OSPCRE-274